<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Control_Dialog</class>
 <widget class="QDialog" name="Control_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="fen">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>220</y>
        <width>75</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>报警器</string>
      </property>
     </widget>
     <widget class="QPushButton" name="deng2">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>120</y>
        <width>75</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>走廊灯</string>
      </property>
     </widget>
     <widget class="QPushButton" name="deng">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>20</y>
        <width>75</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>卧室灯</string>
      </property>
     </widget>
     <widget class="QPushButton" name="deng3">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>170</y>
        <width>75</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>客厅灯</string>
      </property>
     </widget>
     <widget class="QPushButton" name="deng1">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>70</y>
        <width>75</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>厨房灯</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
